rendering_resource_manager_service/session/management/job_manager.py

```diff
@@ -139,19 +159,21 @@ def create_job(self, job_id, executable, params, environment, reservation_name,
         for module in modules:
             description.executable += 'module load ' + module.strip() + '\n'
         description.executable += executable
-        description.total_physical_memory = 2000
-        description.arguments = params
-        description.queue = settings.SLURM_QUEUE
-        if reservation_name == '':
+        description.arguments = job_information.params
+        if job_information.exclusive_allocation:
+            description.executable = '#SBATCH --exclusive\n'
```

Comment:
is this correct, replacing the executable instead of += ? Order not important (i.e. doesn't need to come before module load)?